### ISSUE TITLE
[GH Request] <ADD A REQUEST TITLE HERE>

### ISSUE THREAD

**Issue by cassiezamparini** [openedx/axim-engineering#1258]
### Firm Name

OpenCraft

### Urgency

High (< 3 days)

### Problem/Request

We're hosting a [Summit to Enhance the Core Contributor Experience](https://openedx.atlassian.net/wiki/spaces/COMM/pages/4448419874/Summit+to+Enhance+the+Core+Contributor+Experience+Call+for+Proposals) in October and would like to request:

1. The date is added correctly to the Working Group Calendar: Oct 21, 2024, from 15:00 to 17:00 UTC ([timezone converter](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Summit+to+Enhance+the+Core+Contributor+Experience&iso=20241021T15&p1=1440&ah=2&am=30)). I seem to have added it but need confirmation it's done correctly. 
2. A meeting link is created for the Calendar event and Confluence page.

Could someone assist me?

### Reasoning

To ensure our meetings adhere to Axim's protocols.

**Comment by cassiezamparini** [comment]
Note: I have invited all the CCs on the existing WG calendar event - so if we can keep it as is that would be a plus. And just add the meeting link.

![Screenshot 2024-09-16 at 11 01 53](https://github.com/user-attachments/assets/d732e8af-ba34-4b59-a033-b4bc80457e9f)

**Comment by bmtcril** [comment]
Hi @cassiezamparini ! I've updated the event in the following ways:

- Changed the time zone from Paris time to UTC 15:00 - 17:00 
- Fixed the Confluence link to the summit page
- Replaced the Zoom meeting with a Google Meet that has automatic recording and transcription turned on

I was unable to add co-host permissions, it looks like Xavier may need to do that if you want to have the meeting artifacts sent to more than one person / allow those extra permissions.

Can you close out this ticket if everything looks good to you?

**Comment by cassiezamparini** [comment]
Thanks @bmtcril. I appreciate the quick response.

Thanks for picking the timezone issue up! The Working Group Calendar is in UTC right? Because I see it shows as 17h00 on the calendar which interferes with the BTR WG:

<img width="244" alt="Screenshot 2024-09-16 at 17 53 15" src="https://github.com/user-attachments/assets/263f99b3-7e47-4237-930c-233373f8e217">

Unless it hasn't updated properly on the WG calendar 🤷‍♀️ 

@antoviaque Could you add co-host permissions to the Summit meeting invite? I'm not able to either.

**Comment by bmtcril** [comment]
Looking at the timezone calculator, UTC 1500 on October 21 should be 1100 US eastern time and 1700 Paris time, which does conflict with the BTR meeting. Does the meeting need to be moved to a different time?

**Comment by cassiezamparini** [comment]
@bmtcril Yes, so I've managed to change the time now but please confirm it's correct on your side. It's doing weird things on my side... It should be 15:00 UTC while the BTR is 17:00 UTC.

<img width="223" alt="Screenshot 2024-09-17 at 08 04 45" src="https://github.com/user-attachments/assets/1e490ce5-327c-487c-8637-c038caa3abe3">

**Comment by bmtcril** [comment]
@cassiezamparini I think the confusion may be that once the event is saved the calendar shows the event in your local time zone, which I'm guessing is UTC + 2 on that date? BTR is definitely 1500 - 1600 UTC on October 21, but would show as 1700 for you on Paris time and shows as 1100 for me on New York time. 

To set the CC Experience meeting to 1500 UTC you would need to set it to 1700 in your own time, with the UTC time zone selected and the calendar will display the right time for everyone in the time zone of their own calendar. This does mean that the meeting will overlap BTR, though.

**Comment by cassiezamparini** [comment]
@bmtcril 🙈 sorry about the back and forth on such a simple thing! Could I ask you to double check it on your side now? It seems to be 2 hours before BTR (which is what I want) - so 1pm UTC.

I think my confusion is that the [Working Group Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_v86shrnegshsqgp4fj2k94u7bc@group.calendar.google.com&ctz=Europe/Berlin) is not displaying UTC for me - and I'm almost certain it used to.

**Comment by bmtcril** [comment]
No worries, this is always extremely confusing. Currently I'm seeing the event at the correct time in my time zone, but the event's time zone is UTC + 2 Central Africa time. I don't know that it matters very much as I don't think there are any daylight savings type shifts between now and then that might affect it, but it's definitely odd.

The link to the working group calendar you shared actually has a different time zone embedded in it at the end that is overriding the default: `&ctz=Europe/Berlin` however the default is currently US eastern! I think it's probably correct that the working group calendar itself should be in UTC and will check that assumption with the rest of Axim engineering today and fix it if that's the right thing to do. In the meantime you can change the link to replace `&ctz=Europe/Berlin` with `&ctz=UTC` for that view. In that view the times look correct right now.

**Comment by cassiezamparini** [comment]
> I think it's probably correct that the working group calendar itself should be in UTC

I don't think I ever gave it much thought! As I just copied it over to my calendar - so didn't even think it was EST! I think changing it to UTC would be a good idea. I think that more inline with other practices throughout the community. But I'm not sure.

> In the meantime you can change the link to replace &ctz=Europe/Berlin with &ctz=UTC for that view. In that view the times look correct right now.

Could you assist me here? I can only change it on my personal calendar and then it messes up the time again 😞

**Comment by antoviaque** [comment]
@bmtcril 

> I was unable to add co-host permissions, it looks like Xavier may need to do that if you want to have the meeting artifacts sent to more than one person / allow those extra permissions.

You mean add you and Cassie as co-hosts on the Google Meet? I am not sure where to set this, I rarely ever use Google Meet as a host. I have however reassigned the ownership of the event to you, though, so you don't get restrictions due to the event?

**Comment by bmtcril** [comment]
I think the event is set up mostly correctly now. I will need to add Xavier as a host once Google recognizes the change in owner, which can take a little bit.

**Comment by bmtcril** [comment]
I think this should be done, I'll be at the meeting in case anything comes up last minute
